internal/typeinfo/validate.go

```diff
@@ -72,7 +72,7 @@ func isNil(v reflect.Value) bool {
 	switch v.Kind() {
 	case reflect.Invalid:
 		return true
-	case reflect.Pointer, reflect.Map, reflect.Slice:
+	case reflect.Pointer, reflect.Map:
```

Comment:
It is a little bit weird now that a function that checks whether an element is nil does not care about slices. In my opinion we should change the function name or we should add a explicit check for slices.

Reply:
Changed name to `isInvalidNil`